homeassistant/components/rfxtrx/device_action.py

```diff
@@ -76,7 +76,6 @@ async def async_validate_action_config(
     hass: HomeAssistant, config: ConfigType
 ) -> ConfigType:
     """Validate config."""
-    config = ACTION_SCHEMA(config)
```

Comment:
This method is only called from the device_automation/action.py which already does a more thorough DEVICE_ACTION_SCHEMA validation so this is unnecessary.